<?xml version="1.0"?>
<launch>

    <!-- Load parameters -->
    <rosparam command="load" ns="" file="$(find table_plane_extractor)/config/config.yaml"/>
    <node name="table_plane_extractor_server" pkg="table_plane_extractor" type="table_plane_extractor_srv.py"  output="screen"/>
    <node name="get_objects_on_table" pkg="table_plane_extractor" type="get_objects_on_table.py"  output="screen"/>
    <node name="objects_on_table_visualizer" pkg="table_plane_extractor" type="objects_on_table_vis.py"  output="screen"/>
</launch>